node/src/BaseClient.ts

```diff
@@ -6555,23 +6555,23 @@ export class BaseClient {
     }
 
     /**
-     * Overwrites part of the string stored at `key`, starting at the specified `offset`,
-     * for the entire length of `value`. If the `offset` is larger than the current length of the string at `key`,
-     * the string is padded with zero bytes to make `offset` fit. Creates the `key` if it doesn't exist.
+     * Overwrites part of the GlideString stored at `key`, starting at the specified `offset`,
```

Comment:
I disagree with doc changes. `string` is a [data type](https://redis.io/docs/latest/develop/data-types/strings/) name which redis/valkey stores. We represent it as `GlideString` (`GlideString`, not GlideString).

Reply:
Agree with Yury.  The [SetRange](https://valkey.io/commands/setrange/) command on Valkey does not understand what a `GlideString` data type is, as the type is defined on the Node.js client.

Reply:
I understand the point — does the element is always encodable to string?
If yes — great, leave the string in place, otherwise it might be misleading. 
We might want to use string as it's the data type, but somehow mention that it can be bytes as well.
I don't have the exact wording in mind.

WDYT?

Reply:
We should have a wiki page/paragraph which describes that all user data (including key names) could be a blob (`byte[]`). This applicable for almost all commands, not just 'string' ones - hash field names and values could be binary as well.
Then we can add a link to the that page to the readme and/or into all relevant docs.

-----
**IMO** It is relevant only for commands which operate with bits. 99% of users won't use non-string data for 99% of commands.

Reply:
Agree with both points

Reply:
We are mapping `String | Byte[]` to the Valkey `BulkString` datatype.  That's all we're doing: it needs to be well-defined and simple for the user to understand.  

For the rest of the documentation: this should be defined by the service.  This thin-client only maps arguments and return values for the client language.  Let's leave the actual details of the `setrange` command to the service, and rely on the reference for the user to investigate the finer details.